tests/integration_tests/security/test_jail.py

```diff
@@ -104,6 +104,82 @@ def test_empty_jailer_id(test_microvm_with_api):
         assert expected_err in str(err)
 
 
+def test_exec_file(test_microvm_with_api):
+    """
+    Test the jailer option `--exec-file`
+
+    @type: security
+    """
+    test_microvm = test_microvm_with_api
+
+    # Error case 1: No such file exists
+    pseudo_exec_file_path = "/tmp/pseudo_firecracker_exec_file"
+    if os.path.exists(pseudo_exec_file_path):
+        os.remove(pseudo_exec_file_path)
+    test_microvm.jailer = JailerContext(
+        jailer_id="bd65600d-8669-4903-8a14-af88203add38",
+        exec_file=pseudo_exec_file_path,
+    )
```

Comment:
Doing just:
```suggestion
    test_microvm.jailer.exec_file=pseudo_exec_file_path
```
would work here.